internal/cli/cmd/dac/build/build.go

```diff
@@ -114,13 +119,21 @@ func (o *option) Execute() error {
 	return nil
 }
 
-func (o *option) processFile(file string) error {
-	// NB: most of the work of the `build` command is actually made by the `eval` command of the cue CLI.
-	// NB2: Since cue is written in Go, we could consider relying on its code instead of going the exec way.
-	//      However the cue code is (for now at least) not well packaged for such external reuse.
-	//      See https://github.com/cue-lang/cue/blob/master/cmd/cue/cmd/eval.go#L87
-	// NB3: #nosec is needed here even if the user-fed parts of the command are sanitized upstream
-	cmd := exec.Command("cue", "eval", file, "--out", o.Output, "--concrete") // #nosec
+func (o *option) processFile(file string, extension string) error {
+	var cmd *exec.Cmd
+
+	if extension == goExtension {
+		cmd = exec.Command("go", "run", file) // #nosec
```

Comment:
no way to call the code behind `go run` directly?

Reply:
AFAIK, no. Except if we do a setup as Dagger, [where the CLI is opening a browser](https://github.com/dagger/dagger/blob/f9219591913f76323f83afb8f34d1b75871ee699/cmd/dagger/run.go#L96).

Reply:
Ok, forget about it then